test/acquisition/test_utils.py

```diff
@@ -95,9 +95,12 @@ def test_compute_best_feasible_objective(self):
                             **tkwargs,
                         ).view(-1, 1)
                         if len(batch_shape) > 0:
-                            expected_best_f = expected_best_f.unsqueeze(1)
-                            expected_best_f = expected_best_f.expand(
-                                *sample_shape, *batch_shape, 1
+                            # When `batch_shape = (b,)`, this expands `expected_best_f`
+                            # from shape (3, 1) to (3, 1, 1), then to
+                            # (1, 1, ..., 1, 3, b, 1), where there are
+                            # `len(sample_shape)` leading ones.
```

Comment:
Actually, the previous one was correct
```suggestion
                            # `len(sample_shape) - 1` leading ones.
```